src/views/create-account-view.ts

```diff
@@ -3,11 +3,44 @@ import '../ga-heading';
 import '@material/mwc-textfield';
 import '@material/mwc-button';
 
-import { css, customElement, html, LitElement, TemplateResult } from 'lit-element';
+import { css, customElement, html, LitElement, property, TemplateResult } from 'lit-element';
 
 @customElement('create-account-view')
 export class CreateAccountView extends LitElement
 {
+    @property()
+    userName = '';
+    userMail = '';
+    userPass = '';
+    userSecPass = '';
+
+
+
+    private async createAccount(): Promise<void>
```

Comment:
To maintain some sort of "logical" or "expected" order in files, it would be good to put methods either before or after the rendering. Same goes with properties. We may have to agree on a certain order first.
I personally usually do:

1. rendering
2. observed properties (those that are decorated)
3. non-observed properties
4. internal methods
5. event handler methods